### PR TITLE
Create the "main" component for admin layout

### PR DESCRIPTION
**Pull request by tijmenb**
This component adds the "Layout main" component. It currently only makes the content use the grid (a single column). In the future this component will also contain the flash messages and other page essentials.

https://trello.com/c/4Z5AAIPI/15-create-main-component

### CODE REVIEW

File: app/views/govuk_publishing_components/components/_layout_main.html.erb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<div class="gem-c-layout-main govuk-width-container">
```
Comment (nickcolley) [inline app/views/govuk_publishing_components/components/_layout_main.html.erb]: Does this intend to be the semantic area that the skip-link links too?

In the regular page template we have:

```html
  <div class="govuk-width-container">
    <!-- breadcrumbs, backlink, phasebanner goes here-->
    <main class="govuk-main-wrapper " id="main-content" role="main">
      <h1 class="govuk-heading-xl">Default page template</h1>
    </main>
  </div>
```

https://design-system.service.gov.uk/styles/page-template/

Comment (alex-ju) [reply app/views/govuk_publishing_components/components/_layout_main.html.erb]: I have the main container in the skip link PR coming up. But can be added here too.

File: app/assets/stylesheets/govuk_publishing_components/admin_styles.scss [hunk 2]
```diff
@@ -11,4 +11,5 @@ $govuk-font-url-function: "font-url";
 
 // Components that are only available inside the admin layout
 @import "govuk_publishing_components/components/layout-footer";
-@import "govuk_publishing_components/components/layout-header";
+@import "govuk_publishing_components/components/layout-main";
+@import "govuk_publishing_components/components/layout-header"
```
Comment (alex-ju) [inline app/assets/stylesheets/govuk_publishing_components/admin_styles.scss]: Missing semicolon fails linting

File: app/views/govuk_publishing_components/components/_layout_main.html.erb (hunk 1 above)
Comment (tijmenb) [reply app/views/govuk_publishing_components/components/_layout_main.html.erb]: Hmm, I'm not sure. Let's make this the responsibility of the app at first, after that we'll see how to abstract it into this component.

**Comment by tijmenb** [comment]
Talked about this offline with @alex-ju after https://github.com/alphagov/govuk_publishing_components/pull/414#discussion_r201998780. Closing this PR, we're going to require the applications to set the width container and associated areas themselves. Once we know how that works we might extract it into a component, but we might not need it.

**Pull request closed without merging**